modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/utils/Constants.java

```diff
@@ -40,4 +40,6 @@
      * HTTP content-type text/plain.
      */
     public static final String TEXT_PLAIN = "text/plain";
+
+    public static final String MAP_TYPE = "application/x-www-form-urlencoded";
```

Comment:
This is the HTTP content type that maps to the 'Map' data type of JMS. This is generally accepted as the correct type mapping for JMS Map to HTTP Content type and is the case in WSO2 MB as well as IBM MQ.

https://www.ibm.com/support/knowledgecenter/SSFKSJ_9.0.0/com.ibm.mq.ref.dev.doc/q111120_.htm